src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputNoLineWrapBad.java

```diff
@@ -1,15 +1,17 @@
 package com.puppycrawl.tools.
     checkstyle.checks.whitespace;
 
-import com.puppycrawl.tools.checkstyle.TreeWalker;
+import static java.math.
+	BigInteger.ZERO;
 
-import javax.accessibility.
-    AccessibleAttributeSequence;
+import com.puppycrawl.tools.checkstyle.TreeWalker;
 import java.util.concurrent.atomic.AtomicIntegerFieldUpdater;
+import javax.accessibility.
+		AccessibleAttributeSequence;
```

Comment:
please make a changes to this file as min as possible. You add two lines and all other are changed for now reason.

Please do the same for all Input files your touched.